adagrams/game.py

```diff
@@ -65,21 +68,23 @@ def score_word(word: str) -> int:
     return total_score
 
 
-def get_highest_word_score(word_list: list) -> tuple[str, int]:
-    score_tracker: dict[str, int] = {} 
+def get_highest_word_score(word_list):
+    score_tracker = {} 
     for given_word in word_list:
         score_for_item = score_word(given_word)
         score_tracker[given_word] = score_for_item
     
     winning_score = -1
     winning_word = ""
-    ties: list[str] = []
+    ties = []
 
 # get highest value or add values to ties list
     for word_item, points in score_tracker.items():
         if points > winning_score:
             winning_score = points
             winning_word = word_item
+            if len(ties) > 0:
+                ties.clear
```

Comment:
👀  As written, this code has no effect on the tallying of the ties when a higher score word is found which still results in possible downstream problems. For instance, if I pass
```py
["AAAAAAAAAA", "BBBBBB", "BBBBBBB", "GGGGGGG"]
```
for the words, I should get `"BBBBBBB"` as the winner (score 29), but I get "AAAAAAAAAA" (score 18) as the winner.

The fix for this is small and on this line, but I also encourage you to try tracing through how this particular set of inputs results in a failure.